### ISSUE TITLE
Using contao:migrate to migrate from 4.4 to 4.9 fails

### ISSUE THREAD

**Issue by dmolineus** [contao/contao#2822]
**Affected version(s)**

Contao 4.9

**Description**

When migrating a project from Contao 4.4 to 4.9 the `vendor/bin/contao-console contao:migrate` fails with the error:

```
[2021-03-02 13:33:08] console.ERROR: Error thrown while running command "contao:migrate --no-interaction". Message: "Call to a member function getParameter() on null" {"exception":"[object] (Error(code: 0): Call to a member function getParameter() on null at /xxx/vendor/contao/core-bundle/src/Resources/contao/library/Contao/File.php:140)","command":"contao:migrate --no-interaction","message":"Call to a member function getParameter() on null"} []
```

After scanning the migrations scripts the `Version480Update` seems to cause the issue because it constructs a new instance of `Contao\File` without initializing the contao framework.

https://github.com/contao/contao/blob/1229843fcd1ec18522b85287ee0ebd915a950c18/core-bundle/src/Migration/Version408/Version480Update.php#L167

This leads to the error here:

https://github.com/contao/contao/blob/1229843fcd1ec18522b85287ee0ebd915a950c18/core-bundle/src/Resources/contao/library/Contao/File.php#L140

**Comment by contaoacademy** [comment]
I noticed this error a few days ago. Have I to run the migration again or what is missing now?

**Comment by dmolineus** [comment]
> I noticed this error a few days ago. Have I to run the migration again or what is missing now?

Unfortunately the migration is designed a way that it's not triggered again when the error occurs. Therefore you probably need to check each migration tasks included in the Contao 4.8 migration.

**Comment by contaoacademy** [comment]
I think this fix should be released as soon as possible. All updates from Contao 4.4 to 4.9 will miss the Contao 4.8 migrations 😔
